src/main/kotlin/org/veupathdb/lib/blast/psiblast/PSIBlast.kt

```diff
@@ -11,343 +11,166 @@ interface PSIBlast : BlastQueryWithIPG {
 
```

Comment:
Cleaned up documentation.  Removed the extra text as it is program specific and will be difficult to maintain, while the flags are universal.

I will be adding the full docs to the specific field classes as they are subject to change with the blast+ version.